app/views/profiles/_domain_user.erb

```diff
@@ -0,0 +1,57 @@
+<div class="w-full">
+  <div class="pb-4">
+    <h2 class="text-xl font-semibold leading-tight text-gray-900 dark:text-white">
+      <%= t(:"activerecord.attributes.user.email") %>
+    </h2>
+    <p class="text-gray-500 dark:text-gray-400">
+      <%=@user.email%>
+      <%= t(:'activerecord.attributes.user.missing') if @user.email == nil%>
+    </p>
+    <br/>
+    <h2 class="text-xl font-semibold leading-tight text-gray-900 dark:text-white">
+      <%= t(:"activerecord.attributes.user.provider") %>
+    </h2>
+    <p class="text-gray-500 dark:text-gray-400">
+      <%=@user.provider%>
+      <%= t(:'activerecord.attributes.user.missing') if @user.provider == nil%>
```

Comment:
**nitpick (blocking):** My comment appears to have disappeared, but can this translation key be reverted to what it was previously. `missing` is not an attribute in the `User` model so it doesn't belong in that key structure. It can remain as a page referenced translation key e.g. `profiles.show.missing`

Reply:
Fixed.

Reply:
Thanks, this looks much better!